budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/service/contract/ContractTotalData.java

```diff
@@ -0,0 +1,49 @@
+package org.wickedsource.budgeteer.service.contract;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.joda.money.Money;
+
+
+@AllArgsConstructor
+@NoArgsConstructor
+public class ContractTotalData extends ContractBaseData {
+    private Money budgetSpentGross;
+    private Money budgetGross;
+    private Money budgetLeftGross;
+
+    public ContractTotalData(ContractBaseData baseData)
+    {
+        super();
+        super.setContractAttributes(baseData.getContractAttributes());
+        super.setBudget(baseData.getBudget());
+        super.setBudgetLeft(baseData.getBudgetLeft());
+        super.setBudgetSpent(baseData.getBudgetSpent());
+        super.setTaxRate(baseData.getTaxRate());
+    }
+
+    public void setBudgetSpentGross(Money budget) {
```

Comment:
You do not need to generate getters and setters yourself. You can simply add these annotations to the top of the class and Lombok will do it for you.
```
@Data
@EqualsAndHashCode(callSuper = true)
```